src/styles/utils/variables.scss

```diff
@@ -0,0 +1,12 @@
+$column_grid_count: 4;
+$starscount: 5;
+$card_width: 200px;
+$font_secondary: #616070;
+$color__font_primary: #060b35;
+$color__font_white: #fff;
+$color__background_secondary: #f3f3f3;
+$color__background_primay: #fff;
+$color__buyButton_secondary: #00acdc;
+$color__links_primary: #060b35;
+$color__links_secondary: #00acdc;
+$color__title_secondary: #34568b;
```

Comment:
Maybe I missed some, but why do you use double underscore?
If you want you variable include 'color' (key or tag) you can simply use color_title_secondary or color-title-secondary ( think it's better because you don't confuse variables with selectors.

Reply:
OK, i fixed that